programs/account-compression/src/instructions/nullify_leaves.rs

```diff
@@ -75,6 +77,7 @@ pub fn process_nullify_leaves<'a, 'b, 'c: 'info, 'info>(
 }
 
 #[inline(never)]
+#[heap_neutral]
```

Comment:
I think this is not necessary because this function is only called within ``process_nullify_leaves`` which is already heap neutral.
What do you think?